### ISSUE TITLE
quick question iassd self trainig

### ISSUE THREAD

**Issue by maxiuw** [darrenjkt/MS3D#18]
Hi for adaptation of uda for iassd.
1. Should i use uda_IA-SSD.yaml as it is or should i add self_trainig parameter wich pseudolabel file?
2. Do I need a pretrained IA-SSD model on source? This is a bit either/or question, or True or False if you prefer (and just checking my understanding): a) if I have a pseudo label file, I don't need a source pretrained model, b) if I dont have a pseudolabel file, I need a pretrained model c) I dont need any of them d) I always need both

**Comment by darrenjkt** [comment]
I find that whether you fine-tune the pre-trained model's weights or train the model from scratch depends on what your source and target data is. 

I find that if your target dataset has less resolution than the source dataset of the pre-trained model, fine-tuning from pre-trained weights can help. But the converse does not seem to help so much. For example, to train a model on waymo as target domain, I did not use any pre-trained model. But with nuscenes as target, I fine-tuned from waymo pre-trained model weights. 

Specific to IA-SSD, I have not experimented with fine-tuning vs training from scratch, you'll have to run some experiments. For IA-SSD, since you have to fix a specific number of sampling points, I find that this makes the model a bit inflexible when adapting across different scan patterns. 

If you wish to train with IA-SSD, you can refer to the format of the yaml files for either fine tuning or training from scratch in target_nuscenes or target_waymo. Make sure to specify the pseudo label path accordingly.

**Comment by maxiuw** [comment]
Thanks!
